ports/templates/ports/builds_filtered_table.html

```diff
@@ -54,12 +54,12 @@ <h4 class="text-secondary">Showing {% if port_name != '' %}results for <i>"{{ po
     </thead>
     {% for build in builds %}
         <tr>
-            <td><a href="/ports/{{ build.port_name }}">{{ build.port_name }}</a></td>
+            <td><a href="{% url 'port_detail' build.port_name %}">{{ build.port_name }}</a></td>
             <td>{{ build.builder_name.name }}</td>
-            <td><a target="_blank" href="{{ build.build_url }}">{{ build.build_id }}</a></td>
-            <td>{{ build.time_start }} </td>
+            <td><a target="_blank" href="https://build.macports.org/builders/ports-{{ build.builder_name.name }}-builder/builds/{{ build.build_id }}">{{ build.build_id }}</a></td>
+            <td>{{ build.time_start|date:"Y-m-d" }}  {{ build.time_start|time:"G:i:s" }} </td>
             <td>{{ build.time_elapsed|time:"H:i:s" }}</td>
-            <td><a target="_blank" href="{{ build.watcher_url }}">{{ build.watcher_id }}</a></td>
+            <td><a target="_blank" href="https://build.macports.org/builders/ports-{{ build.builder_name.name }}-watcher/builds/{{ build.watcher_id }}">{{ build.watcher_id }}</a></td>
```

Comment:
Same as above. I would be tempted to say that both could be displayed in the same column, but then ... I mostly don't want to click on the watcher, I usually want to click on the builder, and it needs to be clear which one is which. Most users don't care about the build number, and would just occasionally want to click on the link. So the text for the link could even be much shorter. Still looking for ideas.